README.md

```diff
@@ -88,9 +88,24 @@ make html # create html documentation
 version-mp2-georchestra
 version-mp2
 ```
----
-**NOTE**
 
-When a stable branch is created, kindly update the `version.txt` with the new stable branch name and the same applies to Mapstore2 submodule
-
----
+## Release procedure
+
+All the changes are first delivered as release candidates, turned into release after user acceptance
```

Comment:
All developments land in master branch first and releases will be created from a stable branch.
As soon as a new major version is ready to be released (developments on master are completed and tested) a new stable branch will be created from master (YYYY.MM.xx, eg. 2022.02.xx).
That naming convention has been agreed because tipiccally a stable branch in mapstore2-georchestra refers to the corresponding one in MapStore.

Each stable branch should always reports inside the `version.txt` file the MapStore stable branch used as well as the corresponding release candidate stable version (eg. 2022.02.xx-RCX-geOrchestra).

Given a stable branch in mapstore2-georchestra, further minor releases of the same series (eg. 2022.02) are delivered from it (eg. 2022.02.01). These should include the update to the latest stable branch of MapStore of the same series.

All the releases are first delivered as release candidates, turned into release after acceptance.